core/player/src/view/plugins/template.ts

```diff
@@ -104,12 +111,17 @@ export default class TemplatePlugin implements ViewPlugin {
       type: NodeType.MultiNode,
       override: false,
       values,
-    };
+    } as Node.MultiNode;
```

Comment:
is there any reason to do `as Node.MultiNode` here if `result` is already typed that way?